app/src/main/java/com/radixdlt/android/data/model/transaction/SendTokensLiveData.kt

```diff
@@ -74,15 +78,15 @@ class SendTokensLiveData @Inject constructor(
             r.subscribeOn(Schedulers.io())
                 .subscribe({
                     if (it is SubmitAtomResultAction) {
-                        if (it.type == SubmitAtomResultAction.SubmitAtomResultActionType.STORED) {
-                            // TODO: Will likely need the atom hid to do this which I now can!!
+//                        if (it.type == SubmitAtomResultAction.SubmitAtomResultActionType.STORED) {
+//                             TODO: Will likely need the atom hid to do this which I now can!!
 //                            val transactionEntity = createTransactionEntity(
 //                                amount, to, payLoad, it, tokenClassReference
 //                            )
 //                            transactionsDao.insertTransaction(transactionEntity)
 //                            postValue(it.type.name)
 //                            return@subscribe
-                        }
+//                        }
```

Comment:
Maybe remove dead code.

Reply:
Although not currently using it, I wan't to keep it as I plan to use it hopefully soon!